gallery/plot_transforms.py

```diff
@@ -119,6 +119,15 @@ def plot(imgs, with_orig=True, row_title=None, **imshow_kwargs):
 blurred_imgs = [blurrer(orig_img) for _ in range(4)]
 plot(blurred_imgs)
 
+####################################
+# GaussianNoise
+# ~~~~~~~~~~~~~
+# The :class:`~torchvision.transforms.GaussianNoise` transform
+# perturbs the input image with gaussian noise.
+noisy = T.GaussianNoise(mean=0.0, sigma=(0.1, 5))
+noisy_imgs = [noisy(orig_img) for _ in range(4)]
+plot(noisy_imgs)
```

Comment:
This gives the following result: https://output.circle-artifacts.com/output/job/34f9c3d8-231e-4748-81b9-cda1724a9f16/artifacts/0/docs/auto_examples/plot_transforms.html#gaussiannoise which does not make sense. Let's reduce the number of transformed images and propose acceptable range of sigmas

Reply:
I think we should not keep an accepted range of sigmas, just let the user decide how much distortion does he want ?

Reply:
Fixed the documentation images in the documentation.